install/install.sh

```diff
@@ -154,12 +159,27 @@ echo ""
 echo "We need some information for the admin account"
 read -e -r -p "Admin username: " -i "admin" admin_name
 read -e -r -p "Admin email: " admin_email
-read -e -r -p -s "Admin password: " admin_password
-read -e -r -p -s "Confirm admin password: " confirm_admin_password
-while [ $admin_password -ne $confirm_admin_password ]; do
+while [$admin_email -z ];do
+   echo "Entered email can't be empty!"
+   echo "Please Re-enter email again ... "
+   read -e -r -p "Admin email: " admin_email
+done 
+read -s -e -r -p  "Admin password: " admin_password
+echo ""
+    read -s -e -r -p  "Confirm admin password: " confirm_admin_password
+while [$admin_password -z ];do
```

Comment:
This can be simplified. Why not keep asking for the admin password, and once that's > 1 character proceed to the confirmation?

Reply:
applied the changes to password and email as well